docs/listeners/api.rst

```diff
@@ -115,9 +115,19 @@ class and enabling it with the `exceptionRenderer` configuration option.
     }
   }
 
-However if you are using CakePHP 3.3's PSR7 middleware feature the ``exceptionRenderer``
+``Note:`` however if you are using CakePHP 3.3's PSR7 middleware feature the ``exceptionRenderer``
```

Comment:
Using `**Note:**` makes lot more sense here.  Also "3.3's" should be changed to "3.3+'s".